apps/frontend/src/pages/auth/auth.tsx

```diff
@@ -7,24 +7,38 @@ import {
 	useAppSelector,
 	useCallback,
 	useLocation,
+	useNavigate,
+	useQuery,
 } from "~/libs/hooks/hooks.js";
 import { actions as authActions } from "~/modules/auth/auth.js";
 import {
+	type EmailDto,
+	type ResetPasswordDto,
 	type UserSignInRequestDto,
 	type UserSignUpRequestDto,
 } from "~/modules/users/users.js";
 
-import { SignInForm, SignUpForm } from "./libs/components/components.js";
+import {
+	ForgotPasswordForm,
+	ResetPasswordForm,
+	SignInForm,
+	SignUpForm,
+} from "./libs/components/components.js";
+import { getFormHeader } from "./libs/helpers/helpers.js";
 import styles from "./styles.module.css";
 
 const Auth: React.FC = () => {
-	const dispatch = useAppDispatch();
 	const { pathname } = useLocation();
 
 	const { user } = useAppSelector(({ auth }) => ({
 		user: auth.user,
 	}));
 
+	const token = useQuery().get("token");
```

Comment:
```suggestion
	const { token } = useQuery();
```
Can we do like that?